library/datasort.h

```diff
@@ -81,8 +81,10 @@ struct datasort_cfg {
 	struct list_head		sorted_chunks;
 	/* Result of mergesort */
 	struct datasort_chunk		*result;
-	/* Datasort directory */
-	char				*dir;
+	/* Datasort directory - chunks are merged into single sorted blob here */
+	char				*datasort_dir;
+	/* Chunks directory - if not defined, then chunks are stored in datasort_dir */
+	char				*chunks_dir;
```

Comment:
Name of chunk directory contains backend_id and blob id. It is necessary to guarantee uniqueness and to understand exactly what blob is sorted, because multiple different backends may start datasort simultaneously on _single_ special mounted filesystem.